QUEST.tsv

```diff
@@ -3491,48 +3491,48 @@ QUEST_20151224_003498	{nl}The small boat was unable to withstand the combined we
+QUEST_20151224_003510	Nothing more than a shell of a goddess devoid of any powers or dignity...
+QUEST_20151224_003511	Mage's Writing Table
 QUEST_20151224_003512	There are two letters wedged inside the book.{nl}Do you want to read them?
 QUEST_20151224_003513	It's such a relief that the barrier has been deactivated!{nl}Now we will be able to pray to Zemyna.
-QUEST_20151224_003514	Goddess Zemyna..
-QUEST_20151224_003515	I beg of you to listen to my prayers..
+QUEST_20151224_003514	Goddess Zemyna...
+QUEST_20151224_003515	I beg of you to listen to my prayers...
 QUEST_20151224_003516	Full Soul Stone
 QUEST_20151224_003517	The power that was protecting the sacred object of the religious body disappeared as the thoughts that were confined within the Soul Stone started slipping away.
 QUEST_20151224_003518	An unknown power seems to be blocking the way.
 QUEST_20151224_003519	Pendulum
 QUEST_20151224_003520	The pendulum is moving slowly in a circular pattern.
 QUEST_20151224_003521	There seems to have been a subtle change in the speed at which the pendulum is moving.
-QUEST_20151224_003522	The Pendulum seems to have begun reacting to something.
-QUEST_20151224_003523	The Pendulum seems to have deteced something.
-QUEST_20151224_003524	The Pendulum is revolving quickly.
+QUEST_20151224_003522	The pendulum seems to have begun reacting to something.
+QUEST_20151224_003523	The pendulum seems to have detected something.
+QUEST_20151224_003524	The pendulum is revolving quickly.
 QUEST_20151224_003525	There seems to be something nearby.
 QUEST_20151224_003526	The grave guard's spell doll attacks the sacred object of the religious body.
 QUEST_20151224_003527	Gimmick Scarecrow
 QUEST_20151224_003528	Gimmick Statue
 QUEST_20151224_003529	Everything is becoming clearer now.{nl}I thank you once more.
-QUEST_20151224_003530	My hometown of Delmore Castle.. Is a truly beautiful place.{nl}The neighbors are so kind and the lord is also generous.{nl}
-QUEST_20151224_003531	I was able to receive many rewards and apply for a long vacation thanks to my diligent service as a liason officer.{nl}I was excited at the thought of hosting a feast with the neighbors.{nl}
-QUEST_20151224_003532	But what I saw when I arrived home.. {nl}Was an empty town with nobody in sight.{nl}
-QUEST_20151224_003533	Well, to be completely fair.. It wasn't totally empty. {nl}Demons suddenly appeared from every direction.{nl}
+QUEST_20151224_003530	My hometown of the Delmore Castle... is a truly beautiful place.{nl}The neighbors are so kind and the lord is also generous.{nl}
+QUEST_20151224_003531	I was able to receive many rewards and apply for a long vacation {nl}thanks to my diligent service as a liason officer.{nl}I was excited at the thought of hosting a feast with the neighbors.{nl}
+QUEST_20151224_003532	But what I saw when I arrived home... {nl}It was an empty town with nobody in sight.{nl}
```

Comment:
I think the person is trying to finish their sentence from the line before. Remove the 'it' and keep the 'was' lowercase?